hoomd/md/pair/aniso.py

```diff
@@ -410,16 +481,18 @@ class ALJ(AnisotropicPair):
         * ``epsilon`` (`float`, **required**) - base energy scale
           :math:`\varepsilon` :math:`[energy]`.
         * ``sigma_i`` (`float`, **required**) - the insphere diameter of the
-          first particle type, :math:`[length]`.
+          first particle type, :math:`\sigma_i` :math:`[length]`.
         * ``sigma_j`` (`float`, **required**) - the insphere diameter of the
-          second particle type, :math:`[length]`.
+          second particle type, :math:`\sigma_j` :math:`[length]`.
         * ``alpha`` (`int`, **required**) - Integer 0-3 indicating whether or
           not to include the attractive component of the interaction (see
           above for details).
         * ``contact_ratio_i`` (`float`, **optional**) - the ratio of the contact
-          sphere diameter of the first type with ``sigma_i``. Defaults to 0.15.
+          sphere diameter of the first type with ``sigma_i`` :math:`\beta_i`.
```

Comment:
What does the extra \beta_i mean?

Reply:
`\beta_i` is the math symbol we use in the above equations for the parameter named `contact_ratio_i` in code.

Reply:
Ah, it looked like multiplication or something. I'll move it to the beginning of the line